README.md

```diff
@@ -17,6 +17,37 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 2. Internal-only libraries SHOULD be prefixed with "graze-".
 3. It is RECOMMENDED to follow the style guide mantained in [agis-/git-style-guide](https://github.com/agis-/git-style-guide).
 
+## Logging
+
+1. Logs should be written to stderr and collected to a central store
+2. Application logs should be in [logfmt](https://brandur.org/logfmt) format in development and in logfmt or json for production `-- maybe always in logfmt?`
+
+  ```
+  msg="Request finished" tag=request_finish status=200
+    user=brandur@mutelight.org user_id=1234 app=mutelight app_id=1234
+  ```
+
+3. Logs in logfmt should have the following fields specified by default:
+  1. Generic
+    1. `app` - simple name of the application
+    1. `env` - environment it is running in (`dev`,`test`,`staging`,`integration`,`live`)
+    1. `module` - the area of the code this log is coming from
+    1. `tag` - a unique name associated with this specific log message
+    1. `level` - the log level (preferably using: `debug`,`info`,`error` but `warn`,`fatal` and `panic` are also valid)
+    1. `msg` - a textual description of the message
+    1. `transaction` - a unique transaction id for a particular request (if applicable)
+    1. `ts` - the current timestamp (in RFC3339 format)
```

Comment:
Might be worthwhile specifying that we want the UTC variant (if that's actually what we want - it makes sense to me). An example date would also be nice.